browser/lib/util/http.js

```diff
@@ -3,6 +3,16 @@ var Http = (function() {
 
 	function Http() {}
 
+	function shouldFallback(err) {
+		var statusCode = err.statusCode;
+		/* 400 + no code = a generic xhr onerror. Browser doesn't give us enough
+		 * detail to know whether it's fallback-fixable, but it may be (eg if a
+		 * network issue), so try just in case */
+		return (statusCode === 408 && !err.code) ||
+			(statusCode === 400 && !err.code)      ||
```

Comment:
> hence kludgy

> all except the first line were indented one level relative to the first, which was done with tabs. 

.. which was correct

> (But isn't any longer after the fixup)

Well that's wrong. The number of tabs is defined by indentation level, as you had it, not by the demands of vanity spacing. If your vanity spacing needs are then unsatisfiable, then just invest your aesthetic energy somewhere else :)